web/src/settings_playgrounds.ts

```diff
@@ -15,27 +16,33 @@ import {current_user, realm} from "./state_data";
 import {render_typeahead_item} from "./typeahead_helper";
 import * as ui_report from "./ui_report";
 
+type Playground = {
```

Comment:
This is already defined in `realm_playground.ts` . You can just import the `type`

Reply:
I didn't get you. Isn't `RealmPlayground` type already defined in` realm_playground.ts` ?

Reply:
Oh yeah sorry, I missed that. Fixing it!
Thank you so much